playground/infrastructure/helper.py

```diff
@@ -44,29 +58,194 @@ def find_examples(work_dir: str) -> List[Example]:
         categories:
             - category-1
             - category-2
+    If some example contain beam tag with incorrect format raise an error.
 
     Args:
         work_dir: directory where to search examples.
+        categories_path: path to the file with all supported categories.
 
     Returns:
         List of Examples.
     """
-    # TODO [BEAM-13135] Implement
-    pass
+    failed = False
+    examples = []
+    supported_categories = _get_supported_categories(categories_path)
```

Comment:
your method is called "find examples" and should maximally satisfy this responsibility. Don't need to get categories from file here

Reply:
Changed.